tools/packages.config

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
-    <package id="Cake" version="0.33.0" />
+    <package id="Cake" version="0.34.1" />
```

Comment:
Seems that cake 0.33 can't find VS 2019, so we need to move to 0.34.1. We are not using VS 2019 yet, but this is just so that we don't have to struggle when we do update.